types/three/src/renderers/webxr/WebXR.d.ts

```diff
@@ -1,56 +1,111 @@
+/**
+ * Available session modes
+ */
 export type XRSessionMode = 'inline' | 'immersive-vr' | 'immersive-ar';
 
-export type XRReferenceSpaceType = 'viewer' | 'local' | 'local-floor' | 'bounded-floor' | 'unbounded';
+/**
+ * Reference space types
+ */
+export type XRReferenceSpaceType = 'viewer' | 'local' | 'local-floor' | 'unbounded';
+export type XRBoundedReferenceSpaceType = 'bounded-floor';
 
 export type XREnvironmentBlendMode = 'opaque' | 'additive' | 'alpha-blend';
 
 export type XRVisibilityState = 'visible' | 'visible-blurred' | 'hidden';
 
+/**
+ * Handedness types
+ */
 export type XRHandedness = 'none' | 'left' | 'right';
 
+/**
+ * InputSource target ray modes
+ */
 export type XRTargetRayMode = 'gaze' | 'tracked-pointer' | 'screen';
 
+/**
+ * Eye types
+ */
 export type XREye = 'none' | 'left' | 'right';
 
+/**
+ * Type of XR events available
+ */
 export type XREventType =
+    | 'devicechange'
+    | 'visibilitychange'
     | 'end'
+    | 'inputsourceschange'
     | 'select'
     | 'selectstart'
     | 'selectend'
     | 'squeeze'
     | 'squeezestart'
     | 'squeezeend'
-    | 'inputsourceschange';
+    | 'reset'
+    | 'eyetrackingstart'
+    | 'eyetrackingend';
 
-export type XRAnimationLoopCallback = (time: number, frame?: XRFrame) => void;
+export type XRDOMOverlayType = 'screen' | 'floating' | 'head-locked';
 
-export type XRFrameRequestCallback = (time: number, frame: XRFrame) => void;
+export type XRReflectionFormat = 'srgba8' | 'rgba16f';
 
 export interface XR extends EventTarget {
     requestSession(mode: XRSessionMode, options?: XRSessionInit): Promise<XRSession>;
     isSessionSupported(mode: XRSessionMode): Promise<boolean>;
 }
 
+export interface Constructor<T = object> {
+    new (...args: any[]): T;
+    prototype: T;
+}
```

Comment:
[QUESTION] What is the purpose of this change? Is this a valid change to have in this package?